src/CMakeLists.txt

```diff
@@ -84,7 +84,10 @@ function(valhalla_module)
 
   target_compile_definitions(${library}
     PUBLIC
-      $<$<BOOL:${MSVC}>:VC_EXTRALEAN;WIN32_LEAN_AND_MEAN;NOMINMAX;NOGDI>
+      # ERROR macro clashes with windows.h -> NOGDI necessary (e.g. https://github.com/ipkn/crow/issues/87)
+      # However, NOGDI needs to be undefined in logging.h
+      $<$<BOOL:${WIN32}>:NOGDI;WIN32_LEAN_AND_MEAN>
```

Comment:
seems like these two are the only ones mingw needs

Reply:
Yes, looks good.